src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -3,8 +3,9 @@
 namespace Illuminate\Contracts\Cache;
 
 use Closure;
+use Psr\SimpleCache\CacheInterface;
 
-interface Repository
+interface Repository extends CacheInterface
```

Comment:
I'm doing here the same thing I did in psr-11:51bfc610692c3ed3388dec0ec04bd3baR8